### ISSUE TITLE
The language of text track's cues is not marked programmatically when they are displayed

### ISSUE THREAD

**Issue by OwenEdwards** [videojs/video.js#7487]
## Description
WCAG 2.0/2.1 [Success Criterion 3.1.2 Language of Parts](https://www.w3.org/TR/WCAG21/#language-of-parts) requires that "The human language of each passage or phrase in the content can be programmatically determined ...".

Text tracks (captions, subtitles, and descriptions) may be in any language, and in particular, they may be in a different language from the rest of the content of the page in which they are displayed. Therefore, the text track may not be in the language indicated by the `lang` attribute of the HTML page, or [the `lang` attribute of the player](https://github.com/videojs/video.js/blob/main/src/js/player.js#L804).

Text tracks have a `srclang` attribute which, if present, indicates the language of the text track; this language attribute should be applied as a `lang` attribute either on the whole region that displays the track's cues or on each cue that is displayed.

Specifically, at https://github.com/videojs/video.js/blob/main/src/js/tracks/text-track-display.js#L421, a CSS style is applied to each text track display element according to the track's language; this could also be used to set the `lang` attribute of that element.

## Steps to reproduce
1. Go to https://videojs.com/advanced?video=elephantsdreammp4
2. Play the video
3. Select one of the CC tracks which is not English (e.g. Russian)
4. Pause the video when a caption is being displayed
4. Examine the DOM, and notice that neither the `<div class="vjs-text-track-display" ...>` nor any of its child elements have a `lang` attribute set, although the language has been applied in a CSS class, e.g. `<div class="vjs-text-track-cue vjs-text-track-cue-ru" ...>`
